lib/provider/home_screen_provider.dart

```diff
@@ -64,11 +64,11 @@ class HomeScreenProvider extends ChangeNotifier {
 
   void sortPriceDescending() {
     for (int j = 1; j < productList.length; j++) {
```

Comment:
Use a productList.forEach method to implement the same logic

Reply:
Done